styles.css

```diff
@@ -292,6 +344,99 @@ ul {
   background: #008552;
 }
 
+@media (min-width: 800px) {
+  .work-container {
```

Comment:
Great job @adarachel 🤝

- [ ] Great design. Kindly fix the responsiveness of your desktop version. As the screen size reduces some elements tend to overlap each other. Kindly work on this and your project will be approved.
![Overlap](https://user-images.githubusercontent.com/63917725/193007039-37362f4b-fe22-4f83-8c9d-2dc755389a3a.PNG)